src/components/navBar/navBar.module.scss

```diff
@@ -39,12 +42,21 @@
   color: #87d870;
 }
 
-@media screen and (max-width: 500px) {
+@media screen and (max-width: 600px) {
```

Comment:
What is the reason for proceeding with `600px` here?

Reply:
Hi @harshith-venkatesh , I noticed an issue with Surface Duo earlier which was getting recognised within the 600 px range. Hence this change. I had Surface Duo in my default test devices on chrome and no specific reason.

Surface Duo view before change with 500px in range:
![image](https://user-images.githubusercontent.com/56400876/140916978-5eb92711-fc79-40d7-8ba7-b23e1900fef4.png)

Surface Duo view after change with 600px in range:
![image](https://user-images.githubusercontent.com/56400876/140917117-f8757f16-04f0-4629-a51e-4390ef6c8803.png)

Thanks.